### PR TITLE
Move autoboot steps to autoboot section

### PR DESCRIPTION
**Pull request by VannyBuns**

### CODE REVIEW

File: docs/user-guide/tiramisu/autoboot.md [hunk 1]
```diff
@@ -13,7 +15,14 @@ Currently, every time you want to launch Tiramisu you will have to launch the He
 1. You will be asked if you want to switch the boot title. Press A to select `Switch to PayloadLoader`.
 1. When the process finished, press A to shutdown the console.
 1. The PayloadLoader will now be launched automatically on every boot.
-    - To change the default boot title hold Start (+) on the Gamepad while booting the console. You cannot change the boot title until you set a default environment first.
-    - Navigate the list using the D-Pad to hover on the title you want to autoboot in, then press the Y button to set this to your autoboot title.
-    - Press A to launch the selected title.
 
+We are now going to make the Tiramisu environment start everytime you launch the Health and Safety Information app and select Wii U Menu as the default boot title as well as getting additional homebrew apps.
```
Comment (Lazr1026) [inline docs/user-guide/tiramisu/autoboot.md:19]: copy paste much? :p
```suggestion
We are now going to make the Tiramisu environment start every time you boot the console, as long as set the default boot title to Wii U Menu.
```

File: docs/user-guide/tiramisu/finalizing-setup.md [hunk 2]
```diff
@@ -4,21 +4,11 @@
 
 Now that PayloadLoader, Environment Loader and Tiramisu are installed, we are going to finalize the setup.
 
-We are going to make the Tiramisu environment start automatically when your console autoboots the Health and Safety Information app (or when manually launching it if you chose not to autoboot it) and select Wii U Menu as default as well as getting additional homebrew apps.
-
-### Setting up PayloadLoader, Environment Loader and Tiramisu
+?> Once you're booted into the Tiramisu environment, you can open the Mii Maker at any time to get into the Homebrew Launcher.
```
Comment (Lazr1026) [inline docs/user-guide/tiramisu/finalizing-setup.md:7]:
```suggestion
?> Once you're booted into the Tiramisu environment, you can open the Mii Maker at any time to get into the Homebrew Launcher.
<br>Homebrew Launcher will be empty for first-time modders. Follow the rest of the guide to install apps into the Homebrew Launcher.
<br>To get back into the Mii Maker, simply press the HOME button while in the Homebrew Launcher.
```
so its all in the one info box.

File: docs/user-guide/tiramisu/finalizing-setup.md [hunk 3]
```diff
@@ -4,21 +4,11 @@
...
 
-1. Turn on your Wii U.
-    - The Environment Loader should show up.
-1. Using the D-Pad, navigate to `tiramisu` and press Y to set this to your default environment, then press A to launch into Tiramisu.
-    - You might get a red warning screen telling you that updates aren't blocked properly. Press A to continue anyway. We're going to block updates in the "Blocking Updates" section below.
-    - To open the Environment Loader in the future, you have to hold X while your Wii U is booting up.
-1. On the Tiramisu Boot Selector, the `Wii U Menu` should already be selected, press Y to set this to your default autobooting option, then press A to launch into the Wii U Menu.
-    - To open the Tiramisu Boot Selector in the future, you have hold START (+) while your Wii U is booting up.
+Homebrew Launcher will be empty for first-time modders. Follow the rest of the guide to install apps into the Homebrew Launcher.
```
Comment (Lazr1026) [inline docs/user-guide/tiramisu/finalizing-setup.md:9]:
```suggestion

```

File: docs/user-guide/tiramisu/finalizing-setup.md [hunk 4]
```diff
@@ -4,21 +4,11 @@
...
 
-?> Once you're booted into the Tiramisu environment, you can open the Mii Maker at any time to get into the Homebrew Launcher.
-<br>Homebrew Launcher will be empty for first-time modders. Follow the rest of the guide to install apps into the Homebrew Launcher.
-<br>To get back into the Mii Maker, simply press the HOME button while in the Homebrew Launcher.
+To get back into the Mii Maker, simply press the HOME button while in the Homebrew Launcher.
```
Comment (Lazr1026) [inline docs/user-guide/tiramisu/finalizing-setup.md:11]:
```suggestion

```

**Review by Lazr1026 — CHANGES_REQUESTED** [review]

File: docs/user-guide/tiramisu/autoboot.md (hunk 1 above)
Comment (VannyBuns) [reply docs/user-guide/tiramisu/autoboot.md:19]: Shit I forgot to change that

File: docs/user-guide/tiramisu/finalizing-setup.md (hunk 2 above)
Comment (VannyBuns) [reply docs/user-guide/tiramisu/finalizing-setup.md:7]: Agreed, done

**Pull request closed without merging**